src/routes/index.svelte

```diff
@@ -4,6 +4,8 @@
     import { RouteAccount } from '../utils/routes'
     import { account$ } from '../features/_common/accountStore'
 
+    import AssetView from '../features/intro/intro.svelte'
```

Comment:
You should name it by the component. So `<Intro />` would be correct name. Btw: svelte files should always be uppercase (actually, I'm wondering if this really works...because in the past lowercase components didn't work)

Reply:
done thanks. it worked with lowercase here